browser/actors/FathomChild.jsm

```diff
@@ -30,4 +30,27 @@ class FathomChild extends JSWindowActorChild {
     // TODO: Something like Fathom.runRuleset(this.document);
     console.log(this.contentWindow.location.href);
   }
+
+  addCSSBorderAndLabel(type) {
+    let color = "";
```

Comment:
If we do something like this, we save a mutable variable (less state is great, imo :-)) and, as a bonus, it gets a bit shorter. Plus we avoid a switch statement, which can an attractive hazard (I find it easy to accidentally leave out `break`s):

```js
const COLORS = {shopping: "red",
                article: "blue"};
const color = COLORS[type] || "";
```

Reply:
Good idea, thanks.